docs/getting_started.rst

```diff
@@ -353,6 +353,10 @@ mixins (:class:`~import_export.admin.ImportMixin`,
    A screenshot of the confirm import view.
 
 
+The resource class(es) can be set through `resource_class` parameter.
+It can be either single class or list of classes.
+
+
```

Comment:
I think it would be really great to have a more complete example in the documentation.  Perhaps instead of 'MBook' (which I find is not an intuitive name), you could add something like 'BookSummary', which would be more like a real world example, and would be clearer when people were trying the test application.

I think you could also add an example of how to add a BookSummary to the documentation, which would be a clear way to demonstrate this feature.

Reply:
I extended the documentation a bit and added an example.